### DISCUSSION TITLE
機器學習線上資源分享

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#708] [General]
機器學習與人工智慧引領這個世代，學習機器學習提升自我技能!
這篇文章分享了許多關於機器學習線上課程以及人工智慧工程師有分為四種:
第一種為機器學習工程師；第二種為資料科學家；第三種為自然語言處理科學家；第三種為以人為中心的機器學習設計師。
其實沒想到人工智慧的工作有分那麼多種類，相信未來會出現許多需要維護人工智慧的工作出現而非單純開發。而文章中分享了許多線上課程，我自己最推薦的是goggle的線上課程，goggle 在圖像辨識的部分算是業界的最佳，而且還有推出許多簡易的方式來利用人工智慧解決相關問題，甚至不需要太多程式碼！
快來看看AI工程師在做什麼吧!
https://buzzorange.com/techorange/2022/09/19/how-to-start-career-in-ai/

**Comment by Yunhans** [comment]
雖然看完文章還是不太懂，果然我對AI的造詣還不夠深

**Comment by ruthchen1020** [comment]
希望我以後可以成為其中一個😭

**Comment by danwan1027** [comment]
這樣的分享節省了許多自學者搜尋學習資源的時間，感謝分享！

**Comment by williamyen0623** [comment]
尋找優質的學習資源相當耗時，非常感謝版主的分享～

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
原來AI工程師在做這個

**Comment by Pietra1226** [comment]
感謝資源分享，希望能越來越會使用AI

**Comment by a45320614** [comment]
感謝分享~ Coursera這個線上學習平台上也有不少AI相關的課程資源可供大家學習!

**Comment by BlahFish1006** [comment]
謝謝分享! 希望未來我也能成為這個「21世紀最性感的工作」者 XDD

**Comment by dabbie109303501** [comment]
感謝分享
原來ai工程師有這些種類 還要學會很多種程式
看完蠻佩服自然語言處理科學家的
還要精通語言 根本是資工系與外語學系雙修
